extensions/wikia/ArticleVideo/ArticleVideoController.class.php

```diff
@@ -2,25 +2,24 @@
 
 class ArticleVideoController extends WikiaController {
 	public function featured() {
-		$wg = $this->getApp()->wg;
-		$title = $wg->Title->getPrefixedDBkey();
+		$requestContext = RequestContext::getMain();
+		$title = $requestContext->getTitle()->getPrefixedDBkey();
 
-		$enableArticleFeaturedVideo = ArticleVideoContext::isFeaturedVideoEmbedded( $title );
+		$featuredVideoData = ArticleVideoContext::getFeaturedVideoData( $title );
 
-		if ( $enableArticleFeaturedVideo ) {
-			$wg->Out->addModules( 'ext.ArticleVideo' );
+		if ( !empty( $featuredVideoData ) ) {
+			$requestContext->getOutput()->addModules( 'ext.ArticleVideo' );
 
 			// TODO: replace it with DS icon when it's ready (XW-2824)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)